From b9b4545638ee61fbcdf104fda93f2e28f24bb6d4 Mon Sep 17 00:00:00 2001
From: Allen Xu <allen.xu@freescale.com>
Date: Tue, 17 Apr 2012 14:53:22 +0800
Subject: ENGR00179804 change NAND clock source from pll2_pfd_352M to
 pll2_pfd_400M

Due to pll2_pfd_352M would be used for LVDS, change NAND clock source to
pll2_pfd_400M.

Signed-off-by: Allen Xu <allen.xu@freescale.com>
---
 arch/arm/mach-mx6/clock.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm/mach-mx6/clock.c b/arch/arm/mach-mx6/clock.c
index d83d4e4..b766627 100644
--- a/arch/arm/mach-mx6/clock.c
+++ b/arch/arm/mach-mx6/clock.c
@@ -3874,7 +3874,7 @@ static int _clk_enfc_set_rate(struct clk *clk, unsigned long rate)
 static struct clk enfc_clk = {
 	__INIT_CLK_DEBUG(enfc_clk)
 	 .id = 0,
-	 .parent = &pll2_pfd_352M,
+	 .parent = &pll2_pfd_400M,
 	 .enable_reg = MXC_CCM_CCGR2,
 	 .enable_shift = MXC_CCM_CCGRx_CG7_OFFSET,
 	 .enable = _clk_enable,
-- 
1.7.9.5

